apps/cnspec/cmd/bundle.go

```diff
@@ -129,7 +129,7 @@ var policyLintCmd = &cobra.Command{
 var policyFmtCmd = &cobra.Command{
 	Use:     "format [path]",
 	Aliases: []string{"fmt"},
-	Short:   "Apply style formatting to policy bundles.",
+	Short:   "Apply style formatting to a policy bundle.",
```

Comment:
Didn't know this, so I gave it a try. The command can actually scan multiple bundles:
```
cnspec bundle lint ~/workspace/mondoo/github.com/cnspec-policies/community/ --verbose                                                                            ✔ │ 06:15:20 
→ lint policy bundle file=/home/christian/workspace/mondoo/github.com/cnspec-policies/community/
DBG loading policy bundle file path=/home/christian/workspace/mondoo/github.com/cnspec-policies/community/chef-infra-client.mql.yaml
DBG loading policy bundle file path=/home/christian/workspace/mondoo/github.com/cnspec-policies/community/chef-infra-server.mql.yaml
DBG loading policy bundle file path=/home/christian/workspace/mondoo/github.com/cnspec-policies/community/mondoo-linux-operational-policy.mql.yaml
DBG loading policy bundle file path=/home/christian/workspace/mondoo/github.com/cnspec-policies/community/mondoo-linux-snmp-policy.mql.yaml
DBG loading policy bundle file path=/home/christian/workspace/mondoo/github.com/cnspec-policies/community/mondoo-phoenix-plcnext-security.mql.yaml
→ valid policy bundle
```

Should we keep the text as it was?

Reply:
Good call. Let me fix that tomorrow